packages/components/src/button/style.scss

```diff
@@ -20,43 +20,39 @@
 
 	// Default button style
 	&.is-default {
-		color: #555;
-		border-color: #ccc;
-		background: #f7f7f7;
-		box-shadow: inset 0 -1px 0 #ccc;
+		color: color(theme(button) shade(5%));
```

Comment:
The tricky thing here is that Gutenberg calculates its colors from the various color themes defined in https://github.com/WordPress/gutenberg/blob/master/bin/packages/post-css-config.js. I haven't been able to match the new colors that are defined in core exactly. 

I've tried to get as close as possible without using weird percentage numbers.

Reply:
Personally I think this is fine, as long term we want a more global solution for this, such as generating all the color schemes in the root and applying them globally.